### DISCUSSION TITLE
which version of cwatm_input5min should I use?

### DISCUSSION THREAD

**Discussion by aiinibxl** [iiasa/CWatM#63] [Q&A] [answered]
Hi~ 
I first get access to your FTP server in 2021 and I download ''cwatm_input_5min.zip'' to run my model. Recently I found it was updated in 2022. It seems that there are some differences between these two versions, the waterdemand maps and soil maps are changed, even the ldd map is different.
<img width="254" alt="微信图片_20230305115640" src="https://user-images.githubusercontent.com/96671336/222940843-0e7b7d7f-06bc-4b70-9f91-f430ccad1463.png">
So I wonder why are these changes and which version should I use, look forward to your reply~
Lili

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag,
The input data at 5 minutes have been updated, including the channel network, channel parameters, soil, and demand; an example settings file (settings_CWatM_template_5min.ini) using this updated data is included. One can continue to use the previous data, but we suggest moving to this updated dataset for new runs.
